upgrade-2.x.sh

```diff
@@ -198,6 +198,10 @@ function upgrade_supervisor() {
                     update-resin-supervisor
                     stop_services
                     remove_containers
+                    if version_gt "6.5.9" "${target_supervisor_version}" ; then
```

Comment:
Please try to reference issues, PR or other resources that would show the initial issue and how it was fixed (if at all).